.github/CHANGELOG.md

```diff
@@ -540,6 +540,14 @@
   
 <h3>Breaking changes</h3>
 
+* Deprecated the tape methods `get_resources` and `get_depth` as they are
+  superseded by the `specs` tape attribute.
```

Comment:
```suggestion
* Removed the deprecated tape methods `get_resources` and `get_depth` as they are
  superseded by the `specs` tape attribute.
```

Reply:
I think it is better to be explicit and say we have fully removed them. Deprecating implies it is still there, just marked for removal later :)